Mathlib/Algebra/Group/Equiv/TypeTags.lean

```diff
@@ -113,6 +113,36 @@ and multiplicative endomorphisms of `Multiplicative A`. -/
   { AddMonoidHom.toMultiplicative with
     map_mul' := fun _ _ => rfl }
 
+/-- `Multiplicative (∀ i : ι, K i)` is equivalent to `∀ i : ι, Multiplicative (K i)`. -/
+@[simps]
+def MulEquiv.piMultiplicative (K : ι → Type*) [∀ i, AddZeroClass (K i)] :
```

Comment:
`AddZeroClass` seems stronger than needed.

Reply:
Changed to `Add`.